src/rhsm/connection.py

```diff
@@ -1367,7 +1367,13 @@ def getOwnerList(self, username):
         Returns an owner objects with pem/key for existing consumers
         """
         method = '/users/%s/owners' % self.sanitize(username)
-        return self.conn.request_get(method)
+        owners = self.conn.request_get(method)
+        # BZ 1749395 When a user has no orgs, the return value
+        #  is an array with a single None element.
+        # Ensures the value is the same for a simple None value
+        if owners is None or (len(owners) > 0 and owners[0] is None):
```

Comment:
This has the potential drawback that if `owners` has more than 1 element and the first is `None`, then all the other elements are discarded; I cannot say whether it could actually happen in practice.

Also, a `None` return value of `request_get()` (which it seems it can happen) is now turned into `[]` -- is it wanted?

Maybe a better idea could be to simply filter out `None` values from the `owners` result -- something like:
```python
  if owners is not None:
    owners = [x for x in owners if x is not None]
```